src/index.js

```diff
@@ -330,6 +312,11 @@ Table.defaultProps = {
   remember: {},
   extendColumns: false,
   downloadFn: saveData,
+  borderStyle: {
```

Comment:
Instead of another abstraction, how about we take the actual CSS border attributes? For example: 
```jsx
borderStyle: {
   borderStyle: 'solid',
   borderRadius: 10,
   borderWidth: 2, 
} 
```
This way we can ensure that the component stays compatible with CSS syntax while users don't need to memorize keys for this particular prop.

Reply:
Also, few more suggestions — 1) this will need a mechanism to handle shorthanded border attribute. 2) let's keep border style deactivated by default